readme.md

```diff
@@ -89,12 +97,6 @@ If `bundle install` fails for the GEM `pg`, install `postgresql`:
 
     brew install postgresql
```

Comment:
this will not work on the projects that need postgres 15.

Reply:
that line has not changed. What change would you suggest?